fe/fe-common/pom.xml

```diff
@@ -38,6 +38,14 @@ under the License.
         <starrocks.home>${basedir}/../../</starrocks.home>
     </properties>
 
+    <dependencies>
```

Comment:
our common shouldn't dependt on hadoop-common

Reply:
@kangkaisen can I add another package like `hadoop-ext`, instead of put code in `fe-common`?

Reply:
> @kangkaisen can I add another package like `hadoop-ext`, instead of put code in `fe-common`?

Or just put code into `fe-core`?